src/ui/scroll-window.js

```diff
@@ -191,6 +196,12 @@ export class ScrollWindow extends Phaser.Scene {
                         this.scroll();
                     }
                 });
+
+                // If refreshing when scrolled down, update to previously scrolled position
+                let deltaY =
+                    (this.curScrollBarY - this.scrollBar.y) /
+                    (remainingScrollBarDist / remainingListDist);
+                this.scroll(deltaY);
```

Comment:
This seems to mess up the text window when examining things. What's the purpose of it?

Reply:
It preserves the spot of the scroll bar when it is refreshed. Needed for quest list because if you were trying to scroll down when enemies were being killed by autoclickers, it would refresh to the top at every kill and it would be impossible with fast kills. I'll throw in a flag to determine whether or not it should preserve it to fix this.

Reply:
Updated! https://github.com/EthanVieira/rs-clicker/pull/62/commits/8074d0484ceb6b37e11d37098e47e3539963a00e